sdk/helper/ldaputil/config.go

```diff
@@ -105,6 +114,19 @@ Default: cn`,
 		"certificate": {
 			Type:        framework.TypeString,
 			Description: "CA certificate to use when verifying LDAP server certificate, must be x509 PEM encoded (optional)",
+			DisplayAttrs: &framework.DisplayAttributes{
+				Name: "CA certificate",
+			},
+		},
+
+		"client_tls_cert": {
+			Type:        framework.TypeString,
+			Description: "Client certificate to provide to the LDAP server, must be x509 PEM encoded (optional)",
```

Comment:
similar to my comment above, both this and the `client_tls_key` fields need the `DisplayAttrs` object like this:
```
DisplayAttrs: &framework.DisplayAttributes{
  DisplayName: "Client TLS Cert" // or "Client TLS Key"
  EditType: "file",
},
```

Reply:
I've added these attributes, but I noticed that the other attributes don't have `EditType` as well. Probably we should add them (maybe in another PR)

Reply:
great observation! this is because `EditType` is only needed when a field should have a file upload or textarea. but you're right, we should have that documented. i'll add it in a follow up PR.